easyrsa3/easyrsa

```diff
@@ -674,6 +680,94 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
             fi
         fi
 	fi
+
+	# verify_ssl_lib major version number
+	if verify_ssl_lib; then
+		case "$osslv_major" in
+			1|3) : ;;
+			*) die "build-ca ssl lib: $osslv_major"
+		esac
+	else
+		die "build-ca ssl lib: $osslv_major"
+	fi
+
+	# Generate the CA with AES256
+	case "$osslv_major" in
+	# => BEGIN SSL lib version
+
+	# BEGIN SSL V3
+	3)
+		# OSSL3 uses -noenc not -nodes
+		if [ -n "$nopass" ]; then no_password="-noenc"; fi
+
+		# OpenSSL 3 is different
+		unset -v crypto_opts
```

Comment:
`-aes256` did not "_get into my code_", it is part of the current flow.

However, on subsequent testing, it appears that the problem was only with OpenSSL 3.0.0 because testing with 3.0.1 does not require `crypto_opts` to be unset.

Reply:
I'll have to double check this because I think it should be `opts`.

Reply:
Double checked, this does not have to be unset for OpenSSL 3.0.1

Reply:
Follow-up patch does not have this code.